README.md

```diff
@@ -1,61 +1,131 @@
+management](https://juju.is/beyond-configuration-management) to allow much
+more agile operations for complex cloud workloads.
+
+Shared, open source operators **take infra as code to the next level** with
+community-driven ops and integration code. Reuse of ops code [improves
+quality](https://juju.is/ops-code-quality) and encourages wider community
+engagement and contribution. Operators also improve security through
+consistent automation. Juju operators are a [community-driven
+devsecops](https://juju.is/devsecops) approach to open source operations.
+
+Juju implements the Kubernetes operator pattern, but is also a **universal
+OLM** that extends the operator pattern to traditional applications (without
+Kubernetes) on Linux and Windows. Such machine operators can work on bare
+metal, virtual machines or cloud instances, enabling [multi cloud and hybrid
+cloud operations](https://juju.is/multi-cloud-operations). Juju allows you
+to embrace the operator pattern on [both container and legacy
+estate](https://juju.is/universal-operators). An operator for machine-based
+environments can share 95% of its code with a Kubernetes operator for the
+same app.
+
+**Juju excels at application integration**. Instead of simply focusing on
+lifecycle management, the Juju OLM provides a [rich application graph
+model](https://juju.is/model-driven-operations) that tells operators how to
+integrate with one another. This dramatically simplifies the operations of
+large deployments.
+
+A key focus for Juju is to **simplify operator design, development and
+usage**.  Instead of making very complex operators for specific scenarios,
+Juju encourages devops to make composable operators, each of which drives a
+single Docker image, and which can be reused in different settings.
+[Composable operators](https://juju.is/integration) enable very rich
+scenarios to be constructed out of simpler **operators that do one thing and
+do it well**.
+
+The OLM provides a central mechanism for operator instantiation,
+configuration, upgrades, integration and administration. The OLM provides a
+range of [operator lifecycle services](https://juju.is/operator-services)
+including leader election and persistent state. Instead of manually
+deploying and configuring operators, the OLM manages all the operators in a
+model at the direction of the administrator.
+
+## Open Operator Collection
+
+A world's [largest collection of operators](https://charmhub.io/) all use
+Juju as their OLM. The Charmhub community emphasizes quality, collaboration
+and consistency. Publish your own operator and share integration code for
+other operators to connect to your application.
+
+The [Open Operator Manifesto](https://charmhub.io/manifesto) outlines the
+values of the community and describe the ideal behaviour of operators, to
+shape contributions and discussions.
+
+## Multi cloud and hybrid operations across ARM and x86 infrastructure
```

Comment:
There's a lot of up front Kubernetes text before we reach this section, which seem (to me) to belatedly mention that Juju works in other places as well. Did we want to balance the intro a bit and then dive into the benefits on k8s?

Reply:
Let's emphasize K8s for the moment.